show-available-apps.sh

```
@@ -1,162 +1,111 @@
 #!/bin/bash
 
 # Show Available Applications Script
-# This script displays all available applications organized by category
+# Lists all available applications in the Ubuntu Scripts installer
+# Author: Deepak Nemade (DN)
+# Version: 3.0
 
 # Colors for output
+RED='\033[0;31m'
```

Shellcheck (0.11.0)

[warning] RED appears unused. Verify use (or export if used externally).

(SC2034)